README.md

```diff
@@ -85,6 +85,12 @@ const result = hyphenate(text);
 npm install hyphen
 ```
 
+Install types definitions for Typescript usage.
+
+```
+npm install --save @types/hyphen
```

Comment:
This should be `--save-dev` I think. 😉

Reply:
Sure thing, will fix this, thanks for pointing out!